README.md

```diff
@@ -293,14 +313,14 @@ generate from your probes. It writes those incoming reports as JSON
 lines to a file. Start it like so:
 
 ```
-$ cd collectors/modality-probe-udp-collector
+$ cd ../collectors/modality-probe-udp-collector
```

Comment:
Ditto above, let's assume you're at the root

Reply:
:+1: